tooling/redhat-camel-spring-boot-bom-generator/pom.xml

```diff
@@ -53,7 +53,7 @@
                         </goals>
                         <configuration>
                             <outputDirectory>
-                                ${project.basedir}/../redhat-camel-spring-boot-bom/
+                                ${project.basedir}/target/redhat-camel-spring-boot-bom/
```

Comment:
I think we can replace `${project.basedir}/target` with  `${project.build.directory}`